WORKSPACE

```diff
@@ -2,201 +2,58 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-
-################################################################################
-# Bazel Embedded
-#
-# Contains rules that support building SW for embedded targets. Specifically, we
-# maintain a fork to build for RISCV32I.
-################################################################################
-git_repository(
-    name = "bazel_embedded",
-    commit = "fe65a8aca35ae0bae563efd6a29cc14fcb15140d",
-    remote = "https://github.com/lowRISC/bazel-embedded.git",
-    shallow_since = "1639417565 -0800",
-)
-
-load("@bazel_embedded//:bazel_embedded_deps.bzl", "bazel_embedded_deps")
-
+# Bazel Embedded; Bazel platform configuration for embedded environments.
```

Comment:
We should also say:
```
workspace(name = "opentitan")
```

(According to https://docs.bazel.build/versions/main/skylark/lib/globals.html#workspace, we maybe should set our name to "com_github_lowRISC_opentitan" or "org_opentitan" or somesuch).

Reply:
How about lowrisc_opentitan?